packages/tsconfig.options.json

```diff
@@ -13,7 +13,7 @@
     "module": "es6",
     "moduleResolution": "node",
     "composite": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
@madou: How about this for excluding helpers like `__assign` to reduce bundlesize?

Reply:
how much does it reduce by

Reply:
unsure tbh because it will leave around const let and stuff which are potential de-op paths

what if we just not use spread/rest for now?

Reply:
Yeah yeah. I am removing `es6` and targeting `es5` only and bringing this change
```ts
<style data-c={bucket} /> 
```